### PR TITLE
update(Crafting): New Guns

### PR DESCRIPTION
**Pull request by Savage912-Official**

### CODE REVIEW

File: resources/[inferno-collection]/inferno-weapons/client.lua [hunk 1]
```diff
@@ -67,7 +67,12 @@ Config.Weapons.Single = {
 	"WEAPON_MARKSMANRIFLE_MK2",
 	"WEAPON_FIREWORK",
 	"WEAPON_STUNGUN",
-	"WEAPON_DBSHOTGUN"
+	"WEAPON_DBSHOTGUN",
```
Comment (minipunch) [inline resources/[inferno-collection]/inferno-weapons/client.lua:70]: we don't use this script anymore FYI so no need to update it anymore in the future

File: resources/usa-crafting/config.lua [hunk 2]
```diff
@@ -490,9 +710,9 @@ Config.Keys = {
     CANCEL = {code = 23, label = 'INPUT_ENTER'}
 }
 
-Config.DEFAULT_CRAFT_DURATION_SECONDS = 8 * 60
+Config.DEFAULT_CRAFT_DURATION_SECONDS = 1 * 10
```
Comment (minipunch) [inline resources/usa-crafting/config.lua:713]: lol let's not forget to undo our debug stuff in the future please 😂

File: resources/usa-crafting/config.lua [hunk 3]
```diff
@@ -490,9 +710,9 @@ Config.Keys = {
...
 
-Config.MAX_FAIL_CHANCE = 0.60 -- max (and initial) chance to fail when crafting
+Config.MAX_FAIL_CHANCE = 0.10 -- max (and initial) chance to fail when crafting
```
Comment (minipunch) [inline resources/usa-crafting/config.lua:715]: same here

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/[inferno-collection]/inferno-weapons/client.lua (hunk 1 above)
Comment (Savage912-Official) [reply resources/[inferno-collection]/inferno-weapons/client.lua:70]: Oh ok

File: resources/usa-crafting/config.lua (hunk 3 above)
Comment (Savage912-Official) [reply resources/usa-crafting/config.lua]: Fixed

File: resources/usa-crafting/config.lua (hunk 2 above)
Comment (Savage912-Official) [reply resources/usa-crafting/config.lua]: Fixed

**Review by minipunch — APPROVED** [review]

**Pull request merged**